src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ObjectFormatting/ObjectFormatter.cs

```diff
@@ -40,6 +43,11 @@ private static class Internal
 
         public static string FormatObject(ObjectFormatterFunc formatterFunc, object obj, FormatSpecifier formatSpecifier = FormatSpecifier.None)
         {
+            if ((formatSpecifier & FormatSpecifier.NoSideEffects) != 0)
```

Comment:
So to be clear, the idea is that if someone specifies nfe we will error out since we cannot evaluate that way?